go.mod

```diff
@@ -139,31 +139,31 @@ require (
 )
 
 replace (
```

Comment:
Why do we have all these kube deps?  And why are they all pinned ?

Reply:
`k8s.io/api` and `k8s.io/apimachinery` are used in the webhook. `k8s.io/client-go` is used in e2e tests.

Reply:
Also, we use `k8s.io/kubernetes/test/e2e/framework` in e2e which requires these deps to be pinned.